packages/notifi-dapp-example/src/components/HistoryList.tsx

```diff
@@ -59,15 +62,27 @@ export const HistoryList: React.FC<HistoryListProps> = ({
       `}
     >
       {historyItems.length > 0 ? (
-        <div className={`p-6 border-b border-gray-200 border-opacity-20`}>
-          <div className="m-auto font-medium text-base text-notifi-text">
-            Inbox
-          </div>
-          {unreadCount ? (
-            <div className="m-auto text-sm text-notifi-text-light">
-              {unreadCount} unread message{unreadCount > 1 ? 's' : ''}
+        <div
+          className={`p-6 border-b border-gray-200 border-opacity-20 flex flex-row justify-between items-start`}
+        >
+          <div>
+            <div className="m-auto font-medium text-base text-notifi-text">
+              Inbox
             </div>
-          ) : null}
+            {unreadCount ? (
+              <div className="m-auto text-sm text-notifi-text-light">
+                {unreadCount} unread message{unreadCount > 1 ? 's' : ''}
+              </div>
+            ) : null}
+          </div>
+          <div className="flex flex-row gap-2">
+            <div className="text-notifi-text text-sm">Unreads</div>
+            <Toggle
+              disabled={false}
+              checked={!isIncludeRead}
+              onChange={() => setIsIncludeRead(!isIncludeRead)}
```

Comment:
Shall we use `setIsIncludeRead((prev)=>!prev)` to
1. avoid potential racing issue
2. get rid of the use of `isIncludeRead`, so we do not need to expose `isIncludeRead` property.

Reply:
If we don't expose isIncludeRead, how should we set the value for checked, we may need to set up another state for this value and it looks like duplicated work

Reply:
as mention, we can use `setIsIncludeRead((prev)=>!prev)` to do so. the `prev` instance is the current state of `isIncludeRead`.